data/json/mapgen/office_cubical.json

```diff
@@ -65,28 +67,19 @@
             "vehicles": {
                 ".": { "vehicle": "swivel_chair", "status": 0, "chance": 1 }
             },
-            "mapping": {
-                "S": {
-                    "items": { "item": "fridge", "chance": 50 },
-                    "furniture": "f_sink"
-                },
-                "e": {
-                    "items": { "item": "fridge", "chance": 50 },
-                    "furniture": "f_fridge"
-                }
-            },
-            "place_items": [
-                { "x": 21, "y": [10,11], "item": "fridge", "chance": 50, "repeat": 1 },
-                { "x": [20,21], "y": [7,8], "item": "cleaning", "chance": 50, "repeat": 1 },
-                { "x": [10,12], "y": 16, "item": "office", "chance": 80, "repeat": 1 },
-                { "x": [11,13], "y": 20, "item": "office", "chance": 80, "repeat": 1 },
-                { "x": [20,21], "y": 20, "item": "cubical_office", "chance": 60, "repeat": 1 },
-                { "x": [19,20], "y": 17, "item": "cubical_office", "chance": 60, "repeat": 1 },
-                { "x": [19,20], "y": 14, "item": "cubical_office", "chance": 60, "repeat": 1 },
-                { "x": 2, "y": [19,20], "item": "cubical_office", "chance": 60, "repeat": 1 },
-                { "x": [2,3], "y": 17, "item": "cubical_office", "chance": 60, "repeat": 1 },
-                { "x": [3,4], "y": 14, "item": "cubical_office", "chance": 60, "repeat": 1 },
-                { "x": [4,5], "y": [3,6], "item": "novels", "chance": 70, "repeat": 1 }
+            "place_loot": [
+                { "group": "dining",         "chance": 20, "x": [21,21], "y": [10,10], "repeat" : [1,2] },
+                { "group": "fridge",         "chance": 85, "x": [21,21], "y": [11,11], "repeat" : [1,3] },
+                { "group": "cleaning",       "chance": 80, "x": [20,21], "y": [ 7, 8], "repeat" : [1,2] },
+                { "group": "office",         "chance": 80, "x": [10,12], "y": [16,16], "repeat" : [1,3] },
+                { "group": "office",         "chance": 80, "x": [11,13], "y": [20,20], "repeat" : [1,3] },
+                { "group": "cubical_office", "chance": 60, "x": [20,21], "y": [20,20], "repeat" : [1,3] },
+                { "group": "cubical_office", "chance": 60, "x": [19,20], "y": [17,17], "repeat" : [1,3] },
+                { "group": "cubical_office", "chance": 60, "x": [19,20], "y": [14,14], "repeat" : [1,3] },
+                { "group": "cubical_office", "chance": 60, "x": [ 2, 2], "y": [19,20], "repeat" : [1,3] },
+                { "group": "cubical_office", "chance": 60, "x": [ 2, 3], "y": [17,17], "repeat" : [1,3] },
+                { "group": "cubical_office", "chance": 60, "x": [ 3, 4], "y": [14,14], "repeat" : [1,3] },
+                { "group": "novels",         "chance": 70, "x": [ 4, 5], "y": [ 3, 6], "repeat" : [1,2] }
```

Comment:
`novels` should be separated into `book_novel`, `book_holy`.
And do the same in `homebooks`.